src/_sass/tilt.scss

```diff
@@ -73,6 +73,9 @@ $color-blue-lightest: #98E8ED;
 $color-green-light: #70D37B;
 $opacity-light: 0.64;
 
+// Updated 2020-05
+$color-gray: #586E75;
```

Comment:
i think it's asking for trouble to have both $color-gray and $color-grey, and make them different colors

Reply:
Oops, I meant to have this inside header2.scss...